### ISSUE TITLE
Create an MdH-OpenSearch docker container

### ISSUE THREAD

**Issue by lukas-rotsching** [amosproj/amos2023ss02-open-search-meta-data-hub#8]
## User story
1. As a user
2. I need MdHOS to be fully contained in one or more docker containers
3. So that i have an easy way of installing it on any host machine without having to mess around with complex install procedures.

## Acceptance criteria
* Read [OpenSearch Docker](https://opensearch.org/docs/latest/install-and-configure/install-opensearch/docker/)
* Create necessary dockerfiles for our MdHOS and initialize it with everything we need to start

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by motrell** [comment]
First version online. Please let us know, if you need something more

**Comment by Amir-Hussein-OTH** [comment]
I have just cloned the project, and need to run the command docker compose up, right?

**Comment by motrell** [comment]
exact

**Comment by leondaniel22** [comment]
Works fine!
